src/theming/sap_fiori_3.scss

```diff
@@ -108,7 +108,7 @@
 
   /* Input Group */
   --fdInputGroup_Text_Shadow: none;
-  --fdInputGroup_ControlButton_SideBorder: none;
+  --fdInputGroup_ControlButton_SideBorder: solid 0.0625rem transparent;
```

Comment:
For the width you can use a variable. It can be either `--sapElement_BorderWidth` or `--sapField_BorderWidth` for Input fields:
<img width="743" alt="Screen Shot 2021-07-06 at 11 26 17 AM" src="https://user-images.githubusercontent.com/39598672/124626949-20daf280-de4d-11eb-8941-c6f4e908e7ca.png">
<img width="749" alt="Screen Shot 2021-07-06 at 11 26 03 AM" src="https://user-images.githubusercontent.com/39598672/124626954-20daf280-de4d-11eb-968b-e6c9096309f9.png">